pjsip/src/pjsip-ua/sip_inv.c

```diff
@@ -2830,13 +2836,17 @@ PJ_DEF(pj_status_t) pjsip_inv_set_local_sdp(pjsip_inv_session *inv,
             status = pjmedia_sdp_neg_modify_local_offer2(inv->pool, inv->neg,
                                                          inv->sdp_neg_flags,
                                                          sdp);
-        } else
+        } else {
```

Comment:
I would use an approach used in other places: `goto on_return;`, so that the decrement is done at one place.

Reply:
> I would use an approach used in other places: `goto on_return;`, so that the decrement is done at one place.

Yes, I just follow the existing flow, don't want to kind of adding "noise" to the patch here :)